helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/CodeTransformations.java

```diff
@@ -50,9 +50,10 @@
   private static final Pattern FROM_BYTEBUFFER_METHOD_END_PATTERN = Pattern.compile("return DECODER.decode\\s*\\(\\s*b\\s*\\)\\s*;\\s*}");
   private static final Pattern IMPORT_CODECS_PATTERN = Pattern.compile("import org\\.apache\\.avro\\.message\\.([\\w.]+);");
   private static final Pattern AVROGENERATED_ANNOTATION_PATTERN = Pattern.compile(Pattern.quote("@org.apache.avro.specific.AvroGenerated"));
-  private static final Pattern MODEL_DECL_PATTERN = Pattern.compile(Pattern.quote("private static SpecificData MODEL$ = new SpecificData();"));
+  //avro < 1.11 has "private static SpecificData MODEL$", 1.11+ has "private static final SpecificData MODEL$"
+  private static final Pattern MODEL_DECL_PATTERN = Pattern.compile("private\\s+static\\s+(final\\s+)?SpecificData\\s+MODEL\\$\\s*=\\s*new\\s+SpecificData\\s*\\(\\)\\s*;");
```

Comment:
Is there reason to believe that the native codegen will produce an arbitrary number of spaces? If no, then we should make this simpler (and nicer to read) like how it used to be: `"private static (final )?SpecificData MODEL$ = ..."`. OTOH, if the answer is yes, then you should allow for spaces between the `()` as well.